app/help/en/faq.md

```diff
@@ -64,182 +1109,261 @@ https://www.linkedin.com/in/nedscott
+Currently, the Steem and Steemit logos are the same and is free to use. In the future, Steemit, Inc. will have its own logo so that it can be distinguished from Steem. The Steemit logo will be proprietary while Steem and its three S-shaped squiggles will remain open for public use.
 
-Steemit is not a scam. A scam begins with you having to give money to someone else. Steemit is not a middleman to any of the digital points allocations users receive. Steemit doesn't ask you to invest anything.  The Steem blockchain and points system is entirely its own entity, similar to Bitcoin, and it should be treated with a high level of due diligence for anyone looking to leverage its digital points as cryptocurrency.
+<a href="/faq.html#Table_of_Contents_Steemit__Inc">&#11025;</a>
+<a class="anchor" name="Can_I_purchase_official_Steemit_merchandise"></a>
+## Can I purchase official Steemit merchandise?
 
-The Steemit team is extremely proud of what has been accomplished building on top of Steem and they are excited about what they’re continuing to build. Any insinuation that they’re doing something improper is abjectly false. Steemit’s blockchain-based transparency is essentially unprecedented in the history of social media networks, including those valued at tens of billions of dollars. 
+There is no official merchandise yet, but it is easy to make Steemit gear on zazzle.com or similar websites. 
 
-Steem is based on years of research and development into blockchain technology and built by a team with more than 15+ years combined experience working on blockchain protocols and dapps. Steemit is deeply committed to delivering continuous improvement to the platform now and well into the future. 
+Here is a post with instructions from @artist1989:
+https://steemit.com/art/@artist1989/custom-steem-and-steemit-art-and-instructions-on-how-to-create-your-very-own-steem-steemit-gear-t-shirts-coffee-mugs-etc
 
-Given the long term vision and the further growth it expects, Steemit was designed to leverage a blockchain that has deliberate mechanisms in place to prevent unexpected abandonment of the digital points. The blockchain’s built-in processes on its public, transparent ledger ensure that authors are rewarded for their contribution in Steem Power, Steem and Steem Dollars. 
+<a href="/faq.html#Table_of_Contents_Steemit__Inc">&#11025;</a>
+<a class="anchor" name="Did_Steemit__pre_mine__tokens"></a>
+## Did Steemit "pre-mine" tokens?
 
-The founders of Steemit currently own about half of the Steem supply. They are publicly committed to reducing their percentage ownership by distributing Steemit’s digital points as means for covering costs of scaling, operations and signing new users up the platform. 
+The challenge for a blockchain startup company is that they need tokens to sell to get funding for the development. If the blockchain is launched so that anybody can mine as much as they want, founders will get very little and may not be able to fund the development.
```

Comment:
please remove the entirety of this answer except the last paragraph on line 1167.